english/weed/weed.md

```diff
@@ -9,6 +9,7 @@ See you - Love Pink Unicorn
 
 Then you wonder why you're taking advice from a Love Pink Unicorn (after all, these are known to be
 the most untrustworthy breed of unicorn, far worse, even, than the Grim Charcoal Unicorns of Strawgoh).
+However, the Love Pink Unicorn brings happiness to everyone. How can anyone not have a smile on their face after eating one of those yummy pot cookies?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.